cmd/publish-failure-resolver-go/main.go

```diff
@@ -129,6 +144,52 @@ func main() {
 	}
 }
 
+// Allow-listed content provider type.
+const contentProvider = "universal-content"
+
+// Filters out entries originating from Methode OR Wordpress from a list of UUIDs.
+// Used to prevent republishing of entries which are due update.
+func filterAllowListedUUIDs(uuids []string, filterListPath string) ([]string, error) {
```

Comment:
please split getDenylistedUUIDs() ([]string, error) and removeDenylistedUUIDs(republishUUIDs []string, denylistedUUIDs []string) ([]string, error)